vmdb/spec/controllers/ops_controller_spec.rb

```diff
@@ -177,6 +177,27 @@
     end
 
   end
+
+  context "#build_saved_audit_hash" do
+    it "builds an audit hash" do
+      schedule = FactoryGirl.create(:miq_schedule,
+                                    :name        => "test_schedule",
+                                    :description => "old_schedule_desc",
+                                   )
+
+      old_attribs = schedule.attributes
+      schedule.update_attributes(:description => "new_schedule_desc")
+      schedule.save
+
+      audit_hash = controller.send(:build_saved_audit_hash, old_attribs, schedule, false)
```

Comment:
Can you change the test to call the action on the controller?  I don't want the test to call `send` directly on this method.